s3-supported-api.md

```diff
@@ -0,0 +1,119 @@
+# S3 api
+
+## Supported service api
+1.  Lists all your buckets
+    `aws s3 ls`
+
+## Supported operations/api on Bucket
+
+1.  GetBucket (List objects in the specified bucket)  
+    `aws s3 ls s3://<your_bucket>`  
+    `aws s3api list-objects --bucket <your_bucket>`  
+    `aws s3api list-objects-v2 --bucket <your_bucket>`  
```

Comment:
we support this? aws s3api list-objects-v2 --bucket <your_bucket>?

Reply:
yes, we support it